internal/provider/resource_access_list_test.go

```diff
@@ -40,7 +47,7 @@ func TestTimeUnmarshal(t *testing.T) {
 	//msg := `{"lastUpdateDateTime":"2021-08-03T15:20:29Z"}`
 	bodyBytes := []byte(msg)
 	type TestStruct struct{
-		LastUpdateDateTime *time.Time
+		LastUpdateDateTime *string
```

Comment:
I recall this from a past PR for formatting.  I'm guessing the type changes are to ensure we don't bet that same parse error?

Reply:
Yep. I didn't realize there were JSON parsing tests for this, so I am fixing them here.

Reply:
👍🏻